src/Core/Dfe.Complete.Application/Projects/Queries/ListAllProjects/ListAllProjects.cs

```diff
@@ -27,25 +27,26 @@ public async Task<Result<List<ListAllProjectsResultModel>>> Handle(ListAllProjec
                     .ToListAsync(cancellationToken);
                 
                 var filteredProjectList = request.ProjectStatus == ProjectState.Active
-                    ? projectList.Where(p => p.Project.AssignedTo != null)
+                    ? projectList.Where(p => p.Project?.AssignedTo != null)
                     : projectList;
                 
                 var result = filteredProjectList
                     .Skip(request.Page * request.Count).Take(request.Count)
                     .Select(item => new ListAllProjectsResultModel(
-                        item.Establishment.Name,
-                        item.Project.Id,
+                        item.Establishment?.Name,
+                        item.Project!.Id,
                         item.Project.Urn,
                         item.Project.SignificantDate,
                         item.Project.State,
                         item.Project.Type,
                         item.Project.FormAMat,
-                        $"{item.Project.AssignedTo.FirstName} {item.Project.AssignedTo.LastName}",
-                        item.Project.LocalAuthority.Name,
+                        item.Project.AssignedTo != null
+                            ? $"{item.Project.AssignedTo.FirstName} {item.Project.AssignedTo.LastName}"
```

Comment:
There is a full name property on the assignedTo object
```suggestion
                            ? item.Project.AssignedTo.FullName
```